plugins/Jobs.cpp

```diff
@@ -361,13 +377,13 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         // works!
         SQResult result;
         const string& name = request["name"];
-        string safeNumResults = SQ(max(request.calc("numResults"),1)); 
+        string safeNumResults = SQ(max(request.calc("numResults"),1));
         string selectQuery =
             "SELECT jobID, name, data, parentJobID FROM ( "
                 "SELECT * FROM ("
                     "SELECT jobID, name, data, priority, parentJobID "
                     "FROM jobs "
-                    "WHERE state='QUEUED' "
+                    "WHERE ( (state='RUNNING' AND lastRun IS NOT NULL AND current_timestamp > DATETIME(lastRun, timeout) ) OR state='QUEUED') "
```

Comment:
Technically it's fine to use `CURRENT_TIMESTAMP` here because this is just a select, and not part of the command that will be replicated out.  But for safety, we avoid that and use `SCURRENT_TIMESTAMP()` everywhere to avoid the risk of replicating out a query that will produce different results on different slaves.  So, not sure if this is a problem or not, just surprised and creeped me out to see it.